### ISSUE TITLE
Multiple comma-separated function arguments

### ISSUE THREAD

**Issue by ocervell** [pabluk/pygments-promql#7]
Found another small issue with the following query:
```
histogram_quantile(0.9999,sum(irate(metric_histogram{a=~'(1|2|3)',b=~'(1|2|3)',c=~'(1|2|3)',d=~'(1|2|3)'}[1m])) by(a,b,c))
```
<img width="1037" alt="Screenshot 2021-04-26 at 10 37 52" src="https://user-images.githubusercontent.com/9629314/116053959-7e97d500-a67b-11eb-9bed-92b770018f6b.png">

seems that multiple labels from by aggregator are not parsed properly.

**Comment by pabluk** [comment]
Hey @ocervell! thanks for the full query, I'm testing it now and I'll try to find what is happening with those labels.

**Comment by pabluk** [comment]
I figured out the issue here and I merged this fix #8.
Now, the output for that expression is formatted properly using:
```
pygmentize -f png -O "line_numbers=False,style=monokai" -o example.png example.promql
```

![example](https://user-images.githubusercontent.com/161046/116315579-03d0d600-a7b1-11eb-9138-306bd8b046d0.png)

**Comment by pabluk** [comment]
The new release [0.0.7 ](https://github.com/pabluk/pygments-promql/releases/tag/0.0.7)is now available on PyPI https://pypi.org/project/pygments-promql/

Thanks again @ocervell for helping to improve this lexer and please don't hesitate to share any future issue.
